lms/djangoapps/organizations/views.py

```diff
@@ -253,3 +256,117 @@ def courses(self, request, pk):  # pylint: disable=W0613
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class OrganizationsGroupsUsersList(SecureListAPIView):
+    """
+    OrganizationsGroupsUsersList returns a collection of users for a organization group.
+
+    **Example Request**
+
+        GET /api/organizations/{organization_id}/groups/{group_id}/users
+
+        POST /api/organizations/{organization_id}/groups/{group_id}/users
+
+        DELETE /api/organizations/{organization_id}/groups/{group_id}/users
+
+    ### The OrganizationsGroupsUsersList view allows clients to retrieve a list of users for a given organization group
+    - URI: ```/api/organizations/{organization_id}/groups/{group_id}/users```
+    - GET: Returns a JSON representation (array) of the set of User entities
+    - POST: Creates a new relationship between the provided User, Group and Organization
+        * users: __required__, The identifier for the User with which we're establishing relationship
+    - POST Example:
+
+            {
+                "users" : 1,2,3,4,5
+            }
+
+    - DELETE: Deletes a relationship between the provided User, Group and Organization
+        * users: __required__, The identifier for the User for which we're removing relationship
+    - DELETE Example:
+
+            {
+                "users" : 1,2,3,4,5
+            }
+    """
+
+    model = OrganizationGroupUser
+
+    def get(self, request, organization_id, group_id):  # pylint: disable=W0221
+        """
+        GET /api/organizations/{organization_id}/groups/{group_id}/users
+        """
+        queryset = User.objects.filter(organizationgroupuser__group_id=group_id,
+                                       organizationgroupuser__organization_id=organization_id)
+
+        serializer = SimpleUserSerializer(queryset, many=True)
+
+        return Response(serializer.data, status.HTTP_200_OK)
+
+    def post(self, request, organization_id, group_id):
+        """
+        GET /api/organizations/{organization_id}/groups/{group_id}/users
+        """
+        user_ids = request.DATA.get('users')
+        try:
+            user_ids = map(int, filter(None, user_ids.split(',')))
+        except Exception:
+            raise ParseError("Invalid user id value")
+
+        try:
+            group = Group.objects.get(id=group_id, organizations=organization_id)
+        except ObjectDoesNotExist:
+            return Response({
+                "detail": 'Group {} does not belong to organization {}'.format(group_id, organization_id)
+            }, status.HTTP_400_BAD_REQUEST)
+
+        users_added = []
+        for user_id in user_ids:
+            try:
+                user = User.objects.get(id=user_id)
+                OrganizationGroupUser.objects.create(organization_id=organization_id, group=group, user=user)
+            except (ObjectDoesNotExist, IntegrityError):
+                continue
+
+            users_added.append(str(user_id))
```

Comment:
we are creating a comma separated list of id's in response using join method which requires these id's to be string.